mutagen.yml

```diff
@@ -0,0 +1,19 @@
+sync:
```

Comment:
Tego pliku nie powinno być w repo. Widze żę jest w gitignore, ale jak już wcześniej był w repo to musisz go usunąć:

git rm --cached mutagen.yml